<launch>

    <node pkg="s_map" type="detection.py" name="s_map_detection"  output="screen">
        <param name="camera_name" value="front"/>
        <param name="rgb_topic" value="realsense/rgb/image_raw"/>
        <param name="depth_topic" value="realsense/aligned_depth_to_color/image_raw"/>
        <param name="camera_info_topic" value="realsense/rgb/camera_info"/>
    </node>

    <node pkg="s_map" type="mapping.py" name="s_map_mapping"  output="screen">
        <param name="world_frame" value="map"/>
        <param name="camera_info_topic" value="realsense/rgb/camera_info"/>
    </node>

</launch>